src/doc/rustc/src/platform-support.md

```diff
@@ -276,7 +276,7 @@ target | std | host | notes
 `thumbv7a-pc-windows-msvc` | ? |  |
 `thumbv7a-uwp-windows-msvc` | ✓ |  |
 `thumbv7neon-unknown-linux-musleabihf` | ? |  | Thumb2-mode ARMv7a Linux with NEON, MUSL
-`wasm64-unknown-unknown` | * |  | WebAssembly
+`wasm64-unknown-unknown` | ? |  | WebAssembly
```

Comment:
Please consider adding some minimal target documentation for wasm64-unknown-unknown, such as to document the default features it expects in the runtime.

Reply:
I'm not sure what you want added here? I mentioned above that I don't think it makes sense to document features and I would rather remove the features than try to document them, but otherwise I don't know how much to document this or what should be included here.